core/src/main/java/com/orange/ouds/core/component/OudsButton.kt

```diff
@@ -778,6 +820,25 @@ internal fun PreviewOudsButtonWithRoundedCorners(theme: OudsThemeContract) =
         }
     }
 
+@Preview
+@Composable
+@Suppress("PreviewShouldNotBeCalledRecursively")
+private fun PreviewOudsButtonWithIconBadge(@PreviewParameter(OudsButtonWithIconBadgePreviewParameterProvider::class) count: Int) =
+    PreviewOudsButtonWithIconBadge(theme = getPreviewTheme(), count = count)
+
+@Composable
+internal fun PreviewOudsButtonWithIconBadge(theme: OudsThemeContract, count: Int) = OudsPreview(theme = theme) {
+    PreviewEnumEntries<OudsButtonState>(columnCount = 2) {
+        OudsButton(
+            nullableIcon = OudsButtonIcon(Icons.Filled.FavoriteBorder, ""),
+            nullableLabel = null,
+            onClick = {},
+            appearance = OudsButtonAppearance.Default,
```

Comment:
Should we use the `OudsButtonAppearance.Minimal` for the preview as this is the most common case?